identifiers/os_device.txt

```diff
@@ -47,48 +40,36 @@ Network
 Network Audio
 Network Management Device
 Network Scanner
-Network management device
 Onboard Administrator
-PBX
-PDA
 PDU
 PLC
 POS
```

Comment:
@hdm - We've both `POS` and `Point of Sale` for device values.  I think we should move to one or the other. Any opinion from you before I switch from `POS` to `Point of Sale`?